docs/src/man/hosting.md

```diff
@@ -159,7 +159,7 @@ dependencies your package might have. If Documenter is the only dependency, then
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 
 [compat]
-Documenter = "~0.20"
+Documenter = "~0.22.3"
```

Comment:
Thanks for bringing this up! We need to remember to update this. The recommended way is to constrain Documenter to a minor version though (so you'd still get bugfixes automatically):

```suggestion
Documenter = "~0.22"
```

Reply:
Hmm, I tried to commit that suggestion using the GitHub UI but I get

> You do not have permission to push to this repository.

Reply:
> We need to remember to update this.

We can make this a `@eval` block and read the version field in the project file at build time :)

Reply:
> We can make this a `@eval` block and read the version field in the project file at build time :)

Yup, an option. Is there an easy `Pkg` incantation to get the current version?

Reply:
No, but something like
```
version = match(r"^version = \"(\d.\d.\d)\"$"m,
        read(joinpath(root, "Project.toml"), String)).captures[1]
```
works.

Reply:
> No, but something like
> 
> ```
> version = match(r"^version = \"(\d.\d.\d)\"$"m,
>         read(joinpath(root, "Project.toml"), String)).captures[1]
> ```
> 
> works.

I think a solution that uses this would be optimal. I'm inexperienced with Documenter, and don't yet know how to make the toml block splice the command you wrote. Can one of you make these changes? The allow edits from maintainers box is definitely checked.

Reply:
@ryanelandt Hmm, for some reason I can't apply the suggestion either. Could you commit it and then I'll merge this. We can evalify the version in a separate PR.